sections/sprint-3/uppaal.tex

```diff
@@ -96,18 +96,31 @@ \subsection{Updates to the network protocol}\label{subsec:sprint3networkupdate}
 A good side-effect of generating the \uppaal model was that it required some reflection upon the network data format specified in \autoref{subsec:data-format}.
 This resulted in some minor refactoring of the format.
 \\
-First of all, the format for sending a field anchor position had a timestamp as a part of the package. 
+First of all, the format for sending a field anchor position had a timestamp as a part of the package.
 Since the anchors will only be sent once, it does not make sense to timestamp it so this has been removed.
 The updated version of the data format can be found in \autoref{app:network}
 \\
+\autoref{tab:goal-zone} has been updated to be 8 bit and not 16 bit.
```

Comment:
to be 8 bit? hvad er 8 bit? og var der skrevet et sted at den var 16 før? Mener ikke at vi sendte det med før